tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/ReleasePlan.cs

```diff
@@ -28,6 +28,10 @@ public class ReleasePlan
         public List<SDKInfo> SDKInfo { get; set; } = [];
         public string ReleasePlanSubmittedByEmail { get; set; } = string.Empty;
         public bool IsCreatedByAgent { get; set; }
+        public string ActiveSpecPullRequest { get; set; } = string.Empty;
```

Comment:
curious, will we have problems with the sync between the value of this field and the list of APISpecs in the APISpec workitem ?

Reply:
Yes, We will need to set it from release planner. Currently it is not required because I use this field only to push data from agent to release planner. So it's one way data flow.

Reply:
Filed issue https://github.com/Azure/azure-sdk-tools/issues/10831